corrector.rb

```diff
@@ -0,0 +1,15 @@
+class Corrector
+  attr_accessor :name
+
+  def correct_name(name)
+    if name.length >= 10
+      name.capitalize[0..9]
+    else
+      name.capitalize
+    end
+  end
+end
+
+def validate_name
+  @name = @corrector.correct_name(name)
+end
```

Comment:
- [ ] In keeping with the YAGNI (You Ain't Gonna Need It) rule, I would recommend you remove this method from this class, as you aren't (and won't be) using it.